srml/support/src/traits.rs

```diff
@@ -122,23 +243,42 @@ pub trait Currency<AccountId> {
 	/// Deducts up to `value` from the combined balance of `who`, preferring to deduct from the
 	/// free balance. This function cannot fail.
 	///
+	/// The resulting imbalance is the first item of the tuple returned.
```

Comment:
I think it is clearer to explicitly state that the resulting balance must be handled. Same for `slash`, `deposit_into_existing`, `withdraw`, `deposit_creating`, and `slash_reserved`.

"The resulting imbalance is the first item of the tuple returned and must be handled."

Reply:
```suggestion
	/// The resulting imbalance is the first item of the tuple returned and must be handled.
```

Reply:
This is now explained in the `Imbalance` trait docs.